superset/assets/src/visualizations/BigNumber/BigNumber.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import shortid from 'shortid';
 import { XYChart, AreaSeries, CrossHair, LinearGradient } from '@data-ui/xy-chart';
-import { BRAND_COLOR } from '../../modules/colors';
+import { BRAND_COLOR } from '@superset-ui/color';
```

Comment:
noting that we should probably make this settable in the future since most people will never be able to modify `@superset-ui/color`

Reply:
Umm, good point. maybe `ConstantRegistry` in `@superset-ui/core`?